Amazon_Web_Services/RDS/Log-Collection/rds.py

```diff
@@ -48,13 +48,13 @@
         response=client.download_db_log_file_portion(
             DBInstanceIdentifier=dBInstanceIdentifier,
             LogFileName=logFile['LogFileName'],
-            Marker=readMarker,
```

Comment:
This comma also causes the script to terminate at this point.

Reply:
@danreichert @stevenchamales IS this also resolved w/ the above comma-comment?

Reply:
@kb18951452 , yes it is.  This comma shouldn't cause any problems.